easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'cuSPARSELt'
+version = '0.6.0.6'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://docs.nvidia.com/cuda/cusparselt/index.html'
+description = """NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in
+which at least one operand is a sparse matrix"""
+
+toolchain = SYSTEM
+
+local_arch = {'arm64': 'sbsa'}.get(ARCH, ARCH)
```

Comment:
@Micket for me on Grace-Hopper what works here is `local_arch = {'aarch64': 'sbsa'}.get(ARCH, ARCH)`, not sure if it should be replaced or if we should check for both `arm64` and `aarch64`